WordPressKit/EditorSettings.swift

```diff
@@ -0,0 +1,27 @@
+private struct RemoteEditorSettings: Codable {
+    let editor_mobile: String
```

Comment:
That underscore in the name looks odd. I think other models use the standard camel case and then use `CodingKeys` to specify the JSON names.

Reply:
Changed to camelCase.
Using `CodingKeys` seems to be remanence from an older Swift Version (in some cases at least?) 🤔 
Unless we want to map to a completely different property name.

Fixed using a helper from JSONDecoder to parse snake case into camel case. This was already in place 👍

Reply:
🤯 `.convertFromSnakeCase` TIL